pyanaconda/modules/payloads/payload/dnf/dnf.py

```diff
@@ -17,43 +17,35 @@
 # License and may only be used or replicated with the express permission of
 # Red Hat, Inc.
 #
-from pyanaconda.core.dbus import DBus
-from pyanaconda.modules.common.constants.objects import PAYLOAD_DEFAULT
+from dasbus.server.publishable import Publishable
+
 from pyanaconda.modules.payloads.payload.payload_base import PayloadBase
 from pyanaconda.modules.payloads.payload.dnf.dnf_interface import DNFInterface
-from pyanaconda.modules.payloads.payload.dnf.packages.packages import PackagesModule
 
 from pyanaconda.anaconda_loggers import get_module_logger
 log = get_module_logger(__name__)
 
 
-class DNFModule(PayloadBase):
+class DNFModule(PayloadBase, Publishable):
```

Comment:
I think that the class `PayloadBase` should be `Publishable`. It is an abstract class anyway.

Reply:
Fixed.